### ISSUE TITLE
ep. 3 content: github or R markdown?

### ISSUE THREAD

**Issue by jonjab** [UCSBCarpentry/R-markdown#51]
This note:
"***Note** this is github’s markdown “flavor” for headers,"

Is confusing. Are we doing R markdown or github markdown in this lesson.  

I would suggest deleting this or clarifying exactly what syntax is under discussion and what might be different.

**Comment by torwhite** [comment]
This is only in relation to the webpage - it will not be something that needs to be taught. 

We should clarify that it's a note ONLY for learners who are using the website (sans instructor/workshop) to learn Rmarkdown.

**Comment by rcurty** [comment]
I agree it is confusing, but whoever is instructing will have to advise that this will be only an issue if they are looking at the lesson website. Perhaps we should add a note for instructors.

**Comment by torwhite** [comment]
I think this is important note to include for instructors and learners just following the website. We could try to word it better.

**Comment by torwhite** [comment]
moving this to "clean up after workshop"
